README.rst

```diff
@@ -15,12 +15,14 @@ Setup
 
 Start by `creating a project in the Google Developers Console <https://console.developers.google.com>`_.
 
+In the **Authorized JavaScript origins** add the Root URL for your installation::
```

Comment:
Yeah I dont think it should be needed. This would be required if you used something like an AJAX call, since browser CORS require you to accept a domain request.